Example/CommonWallet/Demo/Default/DefaultDemo.swift

```diff
@@ -16,7 +16,7 @@ final class DefaultDemo: DemoFactoryProtocol {
     var completionBlock: DemoCompletionBlock?
 
     func setupDemo(with completionBlock: @escaping DemoCompletionBlock) throws -> UIViewController {
-        let accountId = try IRAccountIdFactory.account(withIdentifier: "julio@demo")
```

Comment:
What's wrong with Julio?)

Reply:
just wanted to have different account ids per demo :)